### ISSUE TITLE
OCP 4.11 support SNO + Workers - test it :-)

### ISSUE THREAD

**Issue by rbo** [RedHat-EMEA-SSA-Team/hetzner-ocp4#222]
In 4.11, we're adding a support to add workers to SNO - the goal is NOT to have 1000 workers + SNO, but a handful of them

**Comment by tomazb** [comment]
I tried this with the latest version - one control plane node and a few workers, but the install could not finish the work. Something extra I need to define for this?

**Comment by rbo** [comment]
Let's try it, devel tree (v 4.12) with:
```
master_count: 1
compute_count: 3
compute_memory_size: 16384
compute_vcpu: 4
```

**Comment by rbo** [comment]
Bootstrap completed...

**Comment by rbo** [comment]
Works, import setting is: `masters_schedulable: true`  (default is true)

@tomazb from my point of view I would close the issue. Do you have any details why your installation does not finished?

**Comment by tomazb** [comment]
Yes, it works with "masters_schedulable: true" which is not how I deploy most of my masters. My personal default is false, of course.

Maybe add a section to say that with a single master you always need this setting to be true. Even with workers.

**Comment by rbo** [comment]
Good point, at least one finding with issue :-)

**Comment by rbo** [comment]
Documenation is available: https://github.com/RedHat-EMEA-SSA-Team/hetzner-ocp4/?tab=readme-ov-file#single-node
